src/Common/tests/System/Net/SslProtocolSupport.cs

```diff
@@ -26,6 +30,13 @@ public static SslProtocols SupportedSslProtocols
                     supported |= SslProtocols.Ssl3;
                 }
 #pragma warning restore 0618
+#if !netstandard
```

Comment:
Same for all of these.  I wonder if we should still be building those for netstandard.